eng/pipelines/linux.yml

```diff
@@ -92,14 +92,15 @@ jobs:
           - _outerloop: true
 
         - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
-          - linuxDefaultQueues: Centos.7.Amd64.Open+RedHat.7.Amd64.Open+Debian.8.Amd64.Open+Ubuntu.1604.Amd64.Open+Ubuntu.1804.Amd64.Open+OpenSuse.42.Amd64.Open+Fedora.28.Amd64.Open
-          - linuxArm64Queues: '\(Ubuntu.1604.Arm64\)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-helix-arm64v8-b049512-20190321153539'
+          - linuxDefaultQueues: Centos.7.Amd64.Open+RedHat.7.Amd64.Open+Debian.8.Amd64.Open+Ubuntu.1604.Amd64.Open+Ubuntu.1804.Amd64.Open+OpenSuse.42.Amd64.Open+\(Fedora.28.Amd64\)ubuntu.1604.amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-28-helix-45b1fa2-20190402012449
```

Comment:
Should we test CI in Fedora 29 instead? In official builds we're already covering 28 and 29, it makes sense to have CI in the newest, no?

Reply:
I was not sure so I decided to keep it as it was. In case of Alpine 39 turn out to be less stable so I moved toward older and more stable version. I don't want to cause troubles on PR submission. 
My current thinking is to keep same version and verify 28 VM -> 28 container. if that looks good we can follow up with version upgrade.